### DISCUSSION TITLE
最新版umijs引入antd后TS报错

### DISCUSSION THREAD

**Discussion by snowrain0722** [umijs/umi#11570] [Q&A]
<img width="1440" alt="截屏2023-08-28 08 25 25" src="https://github.com/umijs/umi/assets/48642074/28120db8-18b3-4ead-89a7-ec71fd667cd9">
多个声明文件包含了此文件，这个问题怎么处理？

**Comment by snowrain0722** [comment]
antd依赖也是

**Comment by fz6m** [comment]
目前此问题只出现在 npm / yarn 的情况，pnpm 没有问题。

将在 #11572 修复，如正常发布，将在下周下个最新版本中解决此问题。
